src/rohini/Rohini_Test1_Program1.java

```diff
@@ -0,0 +1,26 @@
+package rohini;
+
+public class Rohini_Test1_Program1 {
+
+	void findNonRepeatingChar(String Name) {
+	
+		for (int i = 0; i < Name.length(); i++) {
+			boolean flag=true;
+			for (int j = 0; j < Name.length(); j++) {
+              if (i!=j && Name.charAt(i) == Name.charAt(j)) 
```

Comment:
here you can aslo use 
indexOf(ch) ==lastIndexOf(ch) logic to avoid one more for loop